python/tests/test.py

```diff
@@ -0,0 +1,125 @@
+import pytest
+import random
+
+from shaped_bloom_filter import BloomFilter, BloomFilterExtended
+from shaped_bloom_filter.exceptions import (
+    BloomFilterIncorrectConstructorValues,
+    BloomFilterIncorrectInputException,
+)
+
+
+def default_parametrized_bloom_filter(max_elements=10, error_rate=0.01):
+    return BloomFilter(
+        max_elements=max_elements,
+        error_rate=error_rate,
+    )
+
+
+def default_parametrized_bloom_filter_extended():
+    return BloomFilterExtended(
+        max_elements=10,
+        error_rate=0.01,
+    )
+
+
+def test_bloom_filter_creation_no_args():
+    with pytest.raises(BloomFilterIncorrectConstructorValues):
+        BloomFilter()
+
+
+def test_bloom_filter_creation_all_args():
+    with pytest.raises(BloomFilterIncorrectConstructorValues):
+        BloomFilter(
+            max_elements=10,
+            error_rate=0.01,
+            restore_from_serialized=bytes([0, 1, 2, 3]),
+        )
+
+
+def test_bloom_filter_creation_params():
+    default_parametrized_bloom_filter()
+
+
+def test_bloom_filter_creation_serialized():
+    bf = default_parametrized_bloom_filter()
+    serialized = bf.serialize()
+
+    new_bf = BloomFilter(restore_from_serialized=serialized)
+    new_serialized = new_bf.serialize()
+
+    assert serialized == new_serialized
+
+
+def test_bloom_filter_add():
+    bf = default_parametrized_bloom_filter()
+    bf.add(3)
```

Comment:
what is this testing? that add doesn't throw an error? i think we can remove the add test cases cause to truly test their functionality is correct we need to assert that the item was added and that's pretty much what the is_member tests are doing

Reply:
So let's remove this one and all the others that are similar to this one.